From: Soc Virnyl Estela <contact@uncomfyhalomacro.pl>
Date: Sat, 09 Dec 2023 16:30:31 +0800
Subject: Julia uses a static variant of its libuv

Although this patch exists as a sample. This won't be used.
---

diff -ruN julia-1.9.4.orig/Make.inc julia-1.9.4/Make.inc
--- julia-1.9.4.orig/Make.inc	2023-12-08 06:25:53.640851723 +0800
+++ julia-1.9.4/Make.inc	2023-12-08 10:44:21.177558348 +0800
@@ -1116,7 +1116,7 @@
 endif
 
 ifeq ($(USE_SYSTEM_LIBUV), 1)
-  LIBUV := $(LOCALBASE)/lib/libuv-julia.a
+  LIBUV := -luv
   LIBUV_INC := $(LOCALBASE)/include
 else
   LIBUV := $(build_libdir)/libuv.a
